placeholder-material/src/main/java/com/google/accompanist/placeholder/material/PlaceholderHighlight.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.placeholder.material
+
+import androidx.compose.animation.core.AnimationSpec
+import androidx.compose.animation.core.InfiniteRepeatableSpec
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import com.google.accompanist.placeholder.PlaceholderHighlight
+import com.google.accompanist.placeholder.fade
+import com.google.accompanist.placeholder.shimmer
+
+/**
+ * Creates a [PlaceholderHighlight] which fades in an appropriate color, using the
+ * given [animationSpec].
+ *
+ * @sample com.google.accompanist.sample.placeholder.DocSample_PlaceholderFade
+ *
+ * @param animationSpec the [AnimationSpec] to configure the animation.
+ */
+@Composable
+fun PlaceholderHighlight.Companion.fade(
+    animationSpec: InfiniteRepeatableSpec<Float> = infiniteRepeatable(
+        animation = tween(
```

Comment:
Note defaulting an esing? Would expect FOSI?

Reply:
Ah yes, we should use FOSI for the fade.

Reply:
ack